### PR TITLE
add autoupdate configuration for jasmine

### PR DESCRIPTION
**Pull request by slackersoft**

### CODE REVIEW

File: ajax/libs/jasmine/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
       "name": "Rajan Agaskar"
     },
     {
-      "name": "Christian Williams"
+      "name": "Gregg Van Hove"
```
Comment (PeterDaveHello) [inline ajax/libs/jasmine/package.json]: This change is unrelated to this coimmit's commit log...

**Comment by slackersoft** [comment]
Fair enough, I've removed the maintainers change.

**Comment by PeterDaveHello** [comment]
Would you please merge these two commits into only one commit as our document said?

**Comment by slackersoft** [comment]
Sorry again, should be all set now I think.

**Comment by PeterDaveHello** [comment]
Thanks.
